pytorch_lightning/trainer/properties.py

```diff
@@ -555,6 +555,13 @@ def _results(self) -> Optional[ResultCollection]:
         if active_loop is not None:
             return active_loop.results
 
+    def get_loops_state_dict(self) -> Dict[str, Any]:
+        return {
+            "fit_loop": self.fit_loop.state_dict(),
+            "validate_loop": self.validation_loop.state_dict(),
```

Comment:
Consistency

```suggestion
            "validation_loop": self.validation_loop.state_dict(),
```

Reply:
Changed for `validate_loop` to match Trainer function names.